.github/workflows/bench.yml

```diff
@@ -63,7 +75,23 @@ jobs:
         run: |
           sudo apt-get update && sudo apt-get install -y python3-pip
           pip3 install jtbl
-      - name: Parse inputs
+      - name: Set env
+        run: |
+          # Default benchmark settings optimized for Aptos, can be overwritten with `env` input
+          echo "RUST_LOG=debug" | tee -a $GITHUB_ENV
```

Comment:
I think that if we want to run benchmarks we should remove this, as it costs us some efficiency at proving time.